README.md

```diff
@@ -1598,6 +1614,20 @@ Describes different audio type configurations for PerformAudioPassThru.
 |`audioType`|AudioType|True||
 
 
+### CloudAppProperties
+##### Parameters
+
+| Value |  Type | Mandatory | Description | 
+| ---------- | ---------- |:-----------: |:-----------:|
+|`appName`|String|True||
+|`appID`|String|True||
+|`enabled`|Boolean|False|If true, cloud app will be included in HMI RPC UpdateAppList|
+|`authToken`|String|False|Used to authenticate websocket connection on app activation|
+|`cloudTransportType`|String|False|Specifies the connection type Core should use|
+|`hybridAppPreference`|HybridAppPreference|False|Specifies the user preference to use the cloud app version or mobile app version when both are available|
+|`endpoint`|String|False||
```

Comment:
Again, Could add a description for the `endpoint` param if you think it is required